### DISCUSSION TITLE
'FeatureInfo' object is not callable (Model not called)

### DISCUSSION THREAD

**Discussion by shravankoninti** [rwightman/efficientdet-pytorch#282] [General]
Hi,

I am running this kaggle notebook - https://www.kaggle.com/code/shonenkov/training-efficientdet/notebook

when it is calling -- net = EfficientDet(config, pretrained_backbone=False)

TypeError: 'FeatureInfo' object is not callable

Can you please help me fix this? I observed there are comments about how to fix in that code - but nothing helped me after trying all options!

**Comment by skygazer42** [comment]
I  had the same problem

**Comment by megleo** [comment]
I had the same problem
